### PR TITLE
subtitle kwarg to set_titles_and_labels

### LINKED ISSUE #52: Add subtitle capability to `set_titles_and_lables`
Currently, only a main title can be made with the geocat.viz.util function `set_titles_and_labels`. The argument `maintitle` implies that there should be a centered subtitle option, but there is not. The `lefttitle` and `righttitle` arguments are not sufficient for adding a subtitle to the `NCL_skewt_2_2` example in GeoCAT-examples (shown below). It would be helpful to have built in functionality that creates something similar to the following with an option to change the font size, color, weights, etc of all of the possible titles individually.

Labels were created using the following combination of `set_title_and_label` and `matplotlib` functions.
```
# Use geocat.viz utility functions to add labels
gvutil.set_titles_and_labels(ax=ax,
                             xlabel='Temperature (C)',
                             ylabel='P (hPa)',
                             labelfontsize=14)

# Manually add suptitle and subtitle
fig.suptitle('Raob; [Wind Reports]', fontsize=24, y=0.93)
ax.set_title(subtitle, color='darkgoldenrod', fontweight='bold')
```
image

### PR DESCRIPTION
**Pull request by jukent**
## PR Summary
Adds a subtitle keyword argument to set_titles_and_labels by leveraging the fig.suptitle() function
Closes #52

## PR Checklist

**General**
- [x] Make an issue if one doesn't already exist
- [x] Link the issue this PR resolves by adding `closes #XXX` to the PR description where XXX is the number of the issue.
- [x] Add a brief summary of changes to `docs/release-notes.rst` in a relevant section for the next unreleased release. Possible sections include: Documentation, New Features, Bug Fixes, Internal Changes, Breaking Changes/Deprecated
- [x] Add appropriate labels to this PR
- [x] Make your changes in a forked repository rather than directly in this repo
- [x] Open this PR as a draft if it is not ready for review
- [x] Convert this PR from a draft to a full PR before requesting reviewers
- [x] Passes `precommit`. To set up on your local, run `pre-commit install` from the top level of the repository. To manually run pre-commits, use `pre-commit run --all-files` and re-add any changed files before committing again and pushing.

**Documentation**
- [x] Docstrings have been added to all new functions
- [x] Docstrings have been updated with any function changes
- [x] Internal functions have a preceding underscore (`_`) and have been added to `docs/internal_api/index.rst`
- [x] User facing functions have been added to `docs/user_api/index.rst` under their module

**Examples**
- [x] Any new notebook examples added to `docs/examples/` folder
- [x] Clear all notebook cells
- [x] New notebook files added to `docs/examples.rst` toctree
- [x] New notebook files added to new entry in `docs/gallery.yml` with appropriate thumbnail photo in `docs/_static/thumbnails/`

### CODE REVIEW

**Review by kafitzgerald — APPROVED** [review]
Looks good!  

Should we add this to the usage example as well?

**Comment by jukent** [comment]
Sure thing

File: docs/examples/set_titles_and_labels.ipynb [hunk 1]
```diff
@@ -55,6 +55,35 @@
...
+    "fig, ax = plt.subplots()\n",
+    "\n",
+    "# Plot the example data\n",
+    "ax.plot(x, y)\n",
+    "\n",
+    "# Use geocat.viz.util convenience function to set titles and labels\n",
+    "gv.set_titles_and_labels(ax,\n",
+    "                         maintitle=\"Title\",\n",
+    "                         maintitlefontsize=24,\n",
+    "                         leftitle=\"Left title\",\n",
```
Comment (kafitzgerald) [inline docs/examples/set_titles_and_labels.ipynb:85]:
```suggestion
    "                         lefttitle=\"Left title\",\n",
```

**Review by kafitzgerald — CHANGES_REQUESTED** [review]
There's something funky going on still with the example:
https://geocat-viz--185.org.readthedocs.build/en/185/examples/set_titles_and_labels.html

Looks like a minor typo

**Review by kafitzgerald — APPROVED** [review]

**Pull request merged**